pkg/api/server/v1alpha2/records_test.go

```diff
@@ -0,0 +1,206 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/tektoncd/results/pkg/api/server/test"
+	recordutil "github.com/tektoncd/results/pkg/api/server/v1alpha2/record"
```

Comment:
do we have to alias these imports or can we also just go record.FormatName etc?

Reply:
We don't have to, but I did this since we're using the "record" identifier elsewhere and I wanted to make a clearer distinction.